singularity/Singularity.amd_aocl

```diff
@@ -339,91 +352,90 @@ From: fedora:41
-	../src/ 
-	cd ${INSTALLDIR}/casacore/build && make -s -j ${J}
-	cd ${INSTALLDIR}/casacore/build && make install
+    ../src/ 
+    cd ${INSTALLDIR}/casacore/build && make -s -j ${J}
+    cd ${INSTALLDIR}/casacore/build && make install
     cd $INSTALLDIR
     rm -rf $INSTALLDIR/casacore/build
     rm -rf $INSTALLDIR/casacore/src
 
-	#
-	# Install Python-CASAcore
-	#
-	# In case anything was installed previously
-	pip uninstall -y python-casacore
+    #
+    # Install Python-CASAcore
+    #
+    # In case anything was installed previously
+    pip uninstall -y python-casacore
     pip install --no-deps --no-binary python-casacore "python-casacore==3.5.2"
     
-	#
-	# Install Dysco
-	#
-	mkdir -p $INSTALLDIR/dysco/build
-	cd $INSTALLDIR/dysco && git clone https://github.com/aroffringa/dysco.git src
+    #
+    # Install Dysco
+    #
+    mkdir -p $INSTALLDIR/dysco/build
+    cd $INSTALLDIR/dysco && git clone https://github.com/aroffringa/dysco.git src
     cd src && git checkout $DYSCO_VERSION && echo export DYSCO_VERSION=$(git rev-parse --short HEAD) >> $INSTALLDIR/init.sh
-	cd $INSTALLDIR/dysco/build && cmake3 -DCMAKE_INSTALL_PREFIX=$INSTALLDIR/dysco -DCASACORE_ROOT_DIR=$INSTALLDIR/casacore -DPORTABLE=True ../src && make -s -j $J && make install
+    cd $INSTALLDIR/dysco/build && cmake -DCMAKE_INSTALL_PREFIX=$INSTALLDIR/dysco -DCASACORE_ROOT_DIR=$INSTALLDIR/casacore -DPORTABLE=True ../src && make -s -j $J && make install
     cd $INSTALLDIR
     rm -rf $INSTALLDIR/dysco/build
     rm -rf $INSTALLDIR/dysco/src
 
-	#
-	# Install SAGEcal
-	#
-	mkdir -p $INSTALLDIR/sagecal/build
-	cd $INSTALLDIR/sagecal
-	git clone https://github.com/nlesc-dirac/sagecal.git src
-	cd build
-	cmake -DOpenBLAS_LIB=/opt/OpenBLAS/lib64/libopenblas.so -DCMAKE_INSTALL_PREFIX=$INSTALLDIR/sagecal -DLIB_ONLY=True ../src
-	make -j $j
-	make install
-	rm -rf $INSTALLDIR/sagecal/build
-	rm -rf $INSTALLDIR/sagecal/src
-
-	#
-	# Install AOFlagger
-	#
-	mkdir -p ${INSTALLDIR}/aoflagger/build
-	cd ${INSTALLDIR}/aoflagger && git clone https://gitlab.com/aroffringa/aoflagger.git src && cd ${INSTALLDIR}/aoflagger/src && git checkout ${AOFLAGGER_VERSION} && echo export DYSCO_VERSION=$(git rev-parse --short HEAD) >> $INSTALLDIR/init.sh
-	cd ${INSTALLDIR}/aoflagger/build && cmake3 -DCMAKE_INSTALL_PREFIX=${INSTALLDIR}/aoflagger/ -DCASACORE_ROOT_DIR=${INSTALLDIR}/casacore -DBUILD_SHARED_LIBS=ON -DPORTABLE=True ../src
-	cd ${INSTALLDIR}/aoflagger/build && make -s -j ${J}
-	cd ${INSTALLDIR}/aoflagger/build && make install
+    #
+    # Install SAGECal
+    #
+    mkdir -p $INSTALLDIR/sagecal/build
+    cd $INSTALLDIR/sagecal
+    git clone https://github.com/nlesc-dirac/sagecal.git src
+    cd build
+    cmake -DOpenBLAS_LIB=/opt/OpenBLAS/lib64/libopenblas.so -DCMAKE_INSTALL_PREFIX=$INSTALLDIR/sagecal -DLIB_ONLY=True -Wno-dev -DCMAKE_CXX_FLAGS='-g -O3 -fopenmp -ffast-math -lmvec -lm' -DCMAKE_C_FLAGS='-g -O3 -fopenmp -ffast-math -lmvec -lm' ../src
```

Comment:
Same comment about leaving optimisations for a different time (especially `-ffast-math` may not be safe so should be tested beforehand).